pkg/kubelet/apis/podresources/v1alpha1/api.pb.go

```diff
@@ -17,36 +17,23 @@ limitations under the License.
 // Code generated by protoc-gen-gogo. DO NOT EDIT.
 // source: api.proto
 
-/*
```

Comment:
this change is unexpected... what regenerated this?

Reply:
thanks for noticing it. must be one of the tools under hack that was used to correct the CI error. Let me revert this file and see CI error is that.